src/main/java/com/amplitude/Constants.java

```diff
@@ -2,11 +2,16 @@
 
 public interface Constants {
 
-    String API_URL = "https://api2.amplitude.com/2/httpapi";
-    String SDK_PLATFORM = "amplitude-java";
-    String SDK_VERSION = "1.0.0";
+    String  API_URL                 = "https://api2.amplitude.com/2/httpapi";
+    long    NETWORK_TIMEOUT_MILLIS  = 10000;
+    String  SDK_LIBRARY             = "amplitude-java";
+    String  SDK_VERSION             = "1.0.0";
 
-    int MAX_PROPERTY_KEYS = 1024;
-    int MAX_STRING_LENGTH = 1000;
+    int     MAX_PROPERTY_KEYS       = 1024;
+    int     MAX_STRING_LENGTH       = 1000;
+
+    int     GOOD_RES_CODE_START     = 100;
```

Comment:
Rename this to HTTP_STATUS_CONTINUE.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/100